test/integration/targets/vmware_guest_sendkey/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+# Test code for the vmware_guest_sendkey module
+# Copyright: (c) 2017, Diane Wang (Tomorrow9) <dianew@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- when: vcsim is not defined
+  block:
+  - import_role:
+      name: prepare_vmware_tests
+    vars:
+      setup_attach_host: true
+      setup_datastore: true
+      setup_virtualmachines: true
+
+  - name: create a new virtual machine and power on
```

Comment:
Since you do `setup_virtualmachines`, you don't need to create a VM. See: https://github.com/ansible/ansible/blob/devel/test/integration/targets/vmware_guest_boot_facts/tasks/main.yml for an example.

Reply:
Ok, I see, thank you.